### DISCUSSION TITLE
Experimental GlobeView import not working

### DISCUSSION THREAD

**Discussion by akbierly** [visgl/deck.gl#7105] [Q&A] [answered]
```
import {
    _GlobeView as GlobeView
  } from '@deck.gl/core';
```

shows the error: `Module "@deck.gl/core" has no exported member '_GlobeView'`

I am wondering why this is, given that I have a dependency on the latest version of deck.gl (8.7.0). I see that it is labeled as experimental, but on the website example source I see in that it is using 8.7.0 and structures the import exactly the same as mine. Please help!

**Comment by Pessimistress** [comment, marked as answer]
Check your installed version with `yarn list "@deck.gl/core"` or `npm ls "@deck.gl/core"`.

**Reply by akbierly** [reply]
As expected, 8.7.0

**Reply by akbierly** [reply]
@Pessimistress I just updated to 8.8.3, but still get the same error on the import.

**Reply by Pessimistress** [reply]
I would suggest:
- Copy the website example and confirm that you can run it
- Purge your lock file and reinstall dependencies
- Check your local npm environment (e.g. are you installing from a custom registry?)

**Reply by akbierly** [reply]
@Pessimistress I have discovered the discrepancy. I am using typescript for my project, and the typescript modules for @deck.gl/core do not include an export for _GlobeView.

Is it possible for you to add a typescript module for _GlobeView in the near future so that I can use it? 

Alternatively, do you know if there is a way I can prevent the flat maps from repeating? It is causing some weird behavior where my TripsLayer is rendering on multiple North American continents at the same time...
![image](https://user-images.githubusercontent.com/16945234/178016353-9bd317c9-242d-429d-a29a-0c8dd0f49c3e.png)

**Reply by Pessimistress** [reply]
The typescript typings you are using is not maintained by us. See https://deck.gl/docs/get-started/using-with-typescript.

Map repeat - see the documentation of your base map provider.

**Reply by akbierly** [reply]
Thank you!
